CONTRIBUTING.md

```diff
@@ -17,6 +17,11 @@ contain a cropped screenshot of the problem, and a link to the area. Don't assum
 that we will see exactly what you see. If a particular OSM object is an issue,
 the issue should contain the tagging of the object.
 
+## Easy pickings
+
+Some [easy issues](https://github.com/gravitystorm/openstreetmap-carto/search?q=label%3Aeasy&type=Issues) have been selected
```

Comment:
https://github.com/gravitystorm/openstreetmap-carto/issues?q=is%3Aopen+is%3Aissue+label%3Aeasy is a better URL